addons/tagging/functions/fnc_createTag.sqf

```diff
@@ -31,6 +32,9 @@ _tag setObjectTextureGlobal [0, '\z\ace\addons\tagging\UI\tags\' + _color + '\'
 _tag setPosASL _tagPosASL;
 _tag setVectorDirAndUp _vectorDirAndUp;
 
+// Throw a global event for mision makers
+["tagCreated", [_tag, _color, _object, _unit]] call EFUNC(common,globalEvent);
```

Comment:
Yes, QGVAR is something we'll have to do, specially if we switch to CBA events.

The bad thing is that we are not doing it almost anywhere atm.

I'll change it here though